main.go

```diff
@@ -84,11 +84,27 @@ func authorize(s sessions.Store, h http.Handler) http.Handler {
 			return
 		}
 
+		email, ok = session.Values["email"]
+
+		log.Printf("%s auth=success user=%s email=%s redirect=%s\n", logPrefix, openIDUser.(string), email, redirect)
+
 		r.Header.Set("X-Openid-User", openIDUser.(string))
 		h.ServeHTTP(w, r)
 	})
 }
 
+func suffixMismatch(email, emailSuffixString string) bool {
+	emailSuffixes := strings.Split(config.EmailSuffix, ",")
```

Comment:
I think `config.EmailSuffix` be `emailSuffixString`, as we are getting inputs from function params
@halorgium thoughts?